tests/unit/array/test_advance_indexing.py

```diff
@@ -680,7 +680,7 @@ def test_edge_case_two_strings(storage, config_gen, start_storage):
     [
         ('sqlite', None),
         ('weaviate', WeaviateConfig(n_dim=123)),
-        ('annlite', AnnliteConfig(n_dim=123)),
+        # ('annlite', AnnliteConfig(n_dim=123)),
```

Comment:
why?

Reply:
Looking into it. Something with it not closing properly

Reply:
Then it conflicts its own lock in the second reference